eks-distro-base/scripts/eks-d-common

```diff
@@ -376,6 +376,15 @@ function build::common::clean_install() {
     local -r force=${4:-}
     local -r extract_dir="${5:-$NEWROOT}"
     
+
+    if [ ! $shallow ]; then
```

Comment:
Perf improvement for when we are just using yum to install a bunch of packages, this is significantly faster, esp thru qemu, than doing 1 by 1